mobile/src/main/java/ch/epfl/sdp/mobile/application/chess/parser/Combinators.kt

```diff
@@ -118,4 +118,14 @@ object Combinators {
    */
   fun <I, O> Parser<I, O>.repeatAtLeast(count: Int): Parser<I, List<O>> =
       repeat().filter { it.size >= count }
+
+  /**
+   * Return a [Parser] which doesn't contains any null value
+   *
+   * @param I the type of the input.
+   * @param O the type of the output.
```

Comment:
Missing `@receiver` documentation.

```suggestion
   * @param O the type of the output.
   * @receiver the filtered [Parser]
```

Reply:
Fix in https://github.com/epfl-SDP/android/pull/311/commits/a098793e9af0e1ec99e57c290fe3dc1a038e2ea8